internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -574,3 +574,64 @@ func convertCommand(command *manifest.CommandOverride) ([]string, error) {
 	}
 	return out, nil
 }
+
+func convertPublish(p *manifest.PublishConfig) (*template.PublishOpts, error) {
+	if p == nil || p.Topics == nil {
+		return nil, nil
+	}
+	publishers := template.PublishOpts{}
+	// convert the topics to template Topics
+	for _, topic := range p.Topics {
+		t, err := convertTopic(&topic)
+		if err != nil {
+			return nil, err
+		}
+
+		publishers.Topics = append(publishers.Topics, t)
+	}
+
+	return &publishers, nil
+}
+
+func convertTopic(t *manifest.Topic) (*template.Topics, error) {
+	err := validatePubSubName(t.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	w, err := convertTopicWorkers(t.AllowedWorkers)
```

Comment:
We should validate that the worker services conform to the service name validation requirements: namely, that they are all lowercase, start with a letter, and only contain letters, numbers, and hyphens. Check out `internal/pkg/cli/validate.go` for the `basicNameValidation` function.

Reply:
Added a name validation for the workers similar to validate.go to better validate the service names.